src/enlyze/models.py

```diff
@@ -247,6 +247,9 @@ class ProductionRun:
 
     """
 
+    #: The UUID of the production run
+    uuid: UUID
+
     #: The UUID of the appliance the production run was executed on.
     appliance: Appliance
```

Comment:
Not related to this PR but just noticed: the comment is not accurate. The `appliance` property is a complex object, not just the UUID